pkg/services/playlist/playlistimpl/playlist.go

```diff
@@ -5,13 +5,22 @@ import (
 
 	"github.com/grafana/grafana/pkg/services/playlist"
 	"github.com/grafana/grafana/pkg/services/sqlstore/db"
+	"github.com/grafana/grafana/pkg/setting"
 )
 
 type Service struct {
 	store store
 }
 
-func ProvideService(db db.DB) playlist.Service {
+func ProvideService(db db.DB, cfg *setting.Cfg) playlist.Service {
+	if cfg.IsFeatureToggleEnabled("newDBLibrary") {
+		return &Service{
+			store: &sqlxStore{
+				sqlxdb: db.GetDB(),
```

Comment:
calling [Unsafe()](https://github.com/jmoiron/sqlx/blob/28212d434cdd87418ecd0cb81173690ce7ac6ab6/sqlx.go#L289) solves [this](https://github.com/grafana/grafana/pull/52575#pullrequestreview-1065114923) issue
But as a long term solution (in a follow up) maybe we have to consider adding the `playlist_uid` column to the `playlist_item` table.
```suggestion
				sqlxdb: db.GetDB().Unsafe(),
```